package.json

```diff
@@ -41,6 +41,9 @@
     "tap": "^12.7.0",
     "temp": ">= 0.4.0 < 1"
   },
+  "peerDependencies": {
+    "@aws-sdk/client-s3": "^3.0.0"
```

Comment:
Peer dependency will always show a warning if the library doesn't exist, when installing unzipper.  That is not ideal, because many users may not care about s3.

Reply:
I've changed it to optional - does that work? 
Ideally - we could avoid it entirely but it's needed to build the sdk commands. I can see if it's possible to construct them without the library if we need to avoid it entirely.

Reply:
I don't think it's correct to list it as a dev dependency, it is required at runtime. I will see if I can construct the commands without it.

Reply:
It's only required at runtime if you are actually using it.     A big majority of unzipper users will not be using it and therefore we should not have the s3 sdk be installed by default.

peerDependency is signaling that the library is required for unzipper to work (but it's not, only for users that use the s3_v3 function)

devDependency ensures the library is available for tests but otherwise is not going to be installed.   I think the best way is to keep it like that an clarify in readme that if you want to use aws_s3 you will have to ensure that the aws sdk is installed.